### ISSUE TITLE
Using px instead of (r)em in styles.css

### ISSUE THREAD

**Issue by uukelele** [madskristensen/mathtable#1]
You have lots of hard coded pixel values in styles.css, but for responsiveness and accessibility they should be replaced with (r)em equivalents

Btw, 1 rem = 16px
